### ISSUE TITLE
Torque.js open source

### ISSUE THREAD

**Issue by benedekrozemberczki** [toddwschneider/nyc-citibike-data#11]
Could you open-source the Torque.js script?

**Comment by toddwschneider** [comment]
Hi @benedekrozemberczki, I'm not sure exactly what you mean, but if you view source of [this page](https://toddwschneider.com/posts/a-tale-of-twenty-two-million-citi-bikes-analyzing-the-nyc-bike-share-system/), you'll see a link to https://toddwschneider.com/javascripts/citibike.js, which includes the Torque code. The top portion of that file is the torque.js library itself. My code starts at the line `$(function() {`

The only other piece that I can remember is that I used [this repo](https://github.com/CartoDB/torque-gen/) to generate the Torque tilecubes. Unfortunately I don't have the exact command I ran, but it definitely used the torque-gen repo. Sorry this probably doesn't 100% answer your question, but those are all the resources I can remember. Thanks
